packages/standalone-uis/execution-timeline/webpack.config.js

```diff
@@ -86,12 +86,12 @@ module.exports = {
     ]),
   ],
 
-  /*
-   * When using `templateUrl` and `styleUrls` please use `__filename`
-   * rather than `module.id` for `moduleId` in `@View`
-   */
-  node: {
-    crypto: false,
-    __filename: true,
-  },
+  // /*
+  //  * When using `templateUrl` and `styleUrls` please use `__filename`
+  //  * rather than `module.id` for `moduleId` in `@View`
+  //  */
+  // node: {
+  //   crypto: false,
+  //   __filename: true,
+  // },
```

Comment:
Same as above, could we remove it instead?

Reply:
Same as above. I'm not sure this is actually needed, so may remove.